app/views/content/life-as-a-teacher/pay-and-benefits/teacher-pay.md

```diff
@@ -78,8 +78,12 @@ This could be a one-off payment for a specific project, for example, creating a
 
 Or it might be an extra responsibility attached to your job role, for example, being the head of department.
 
+These additional payments are called teaching and learning responsibility (TLR) payments.
```

Comment:
This whole section now feels very spread out - is there any merit in combining a couple of sentences into a para (such as `This could be a one-off.....` and `Or it might be an.....`?

Reply:
But I don't feel strongly about this so up to you!